app/src/main/java/edu/uc/groupproject/covid19tracker/service/CasesByCountryDataProvider.kt

```diff
@@ -35,10 +35,10 @@ class CasesByCountryDataProvider {
                      * data.
                      */
                     val casesByCountry = response.body()!!.string()
-                    val Jobject = JSONObject(casesByCountry)
-                    val Jarray = Jobject.getJSONArray("countries_stat")
-                    for (i in 0 until Jarray.length()) {
-                        val `object` = Jarray.getJSONObject(i).getString(dataType)
+                    val jsonObject = JSONObject(casesByCountry)
+                    val jsonArray = jsonObject.getJSONArray("countries_stat")
+                    for (i in 0 until jsonArray.length()) {
+                        val `object` = jsonArray.getJSONObject(i).getString(dataType)
```

Comment:
Naming conventions (camel case): https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
This came up in a previous code review as well.

I'd take this one step further.  Having a JSONObject variable named jObject is like having a String variable named "string".  Change this to a name that better represents its contents.  And yes, make it lowerCamelCase.